veryfi/__init__.py

```diff
@@ -104,7 +105,7 @@ def _request(self, http_verb, endpoint_name, request_arguments, file_stream=None
                                          timeout=self.timeout)
 
         if response.status_code not in [200, 201, 202, 204]:
-            raise VeryfiClientError(response)
+            raise VeryfiClientError.from_response(response)
```

Comment:
now raise returns neat error message like `BadRequest: 400, Bad or Missing parameters.`, not class str like `VeryfiClientError: <Response [400]>`

Reply:
Good point.